rebar.config

```diff
@@ -14,13 +12,22 @@
             warn_unused_import,
             warnings_as_errors]}.
```

Comment:
I like debug_info defined in the primary options because it enables better debugging with redbug.